LaTexMD/index.md

```diff
@@ -0,0 +1,16 @@
+# Using LaTex in markdown
+
+Since GitHub markdown doesn't natively support LaTex, we use markdeep.  
+Specifically, a slightly modified version of markdeep to look like GitHub markdown.
+
+Using markdeep is relatively straight forward. You write your markdown as you normally would, and use the special markdeep syntax for its extra features (the syntax is explained on [their website](https://casual-effects.com/markdeep/)).  
+Besides that you add the script snippets at the bottom of your markdown file, and save it with the extension `.md.html`.  
+The script snippets can be found in [template.md.html](template.md.html).
+
+# Requirements
+
+1. You are not allowed to use a different style sheet, we want all pages to look uniform across the website.
```

Comment:
Hmm... maybe what about a slightly different wording. instead of `you are not allowed to...` write `we encourage you to use style x`

Reply:
This was actually one of my requirements for the website.  We are aiming for uniformity, which makes it easier to read.

Reply:
How is encouraging a better word when the reality is that we're rejecting PRs that are in a different style?

Reply:
I get scolded for being real on our server, but here its fine?